types/sinon/index.d.ts

```diff
@@ -79,6 +79,7 @@ declare namespace Sinon {
         alwaysReturned(obj: any): boolean;
         invokeCallback(...args: any[]): void;
         getCall(n: number): SinonSpyCall;
+        getCalls(): SinonSpyCall[];
```

Comment:
I don't see any documentation on sinonjs.org for a getCalls() on the spy API, only getCall(). Can you provide a reference for this?

I do see it's included in the Sinon source code, but I don't think that undocumented or private APIs are supposed to be included in typings. So this change might require clarification from the Sinon developers and a doc change before being accepted.

Anyone more deeply involved with DefinitelyTyped please correct me on this if I'm wrong about what should be accepted!